tests/rules/use-selector-in-select.test.ts

```diff
@@ -0,0 +1,99 @@
+import { stripIndent } from 'common-tags'
+import rule, {
+  ruleName,
+  messageId,
+} from '../../src/rules/use-selector-in-select'
+import { ruleTester } from '../utils'
+
+ruleTester().run(ruleName, rule, {
+  valid: [
+    stripIndent`
+    const selectCustomers = createSelector(
```

Comment:
I think all the `createSelectors` can be removed and that it would be enough to write on valid tests as:

```ts
// 1
store.pipe(select(selectCustomers))

// 2
store.pipe(select(selectCustomers))
```

Reply:
You're right, let's keep it simple